<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleEditor</class>
 <widget class="QWidget" name="ConsoleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="fileLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Файл:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="clearFileButton">
       <property name="toolTip">
        <string>Очистить файл</string>
       </property>
       <property name="statusTip">
        <string>Очистить файл</string>
       </property>
       <property name="whatsThis">
        <string>Очистить файл</string>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="text">
        <string>Очистить файл</string>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="configButton">
       <property name="toolTip">
        <string>Настройки</string>
       </property>
       <property name="statusTip">
        <string>Настройки</string>
       </property>
       <property name="whatsThis">
        <string>Настройки</string>
       </property>
       <property name="text">
        <string>Настройки</string>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CustomWidgetsQt::CustomTextEditor" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomWidgetsQt::CustomTextEditor</class>
   <extends>QTextEdit</extends>
   <header>ConsoleEditor/priv/customtexteditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
